src/github_runner_image_builder/cli.py

```diff
@@ -140,6 +133,13 @@ def get_latest_build_id(cloud_name: str, image_name: str) -> None:
     help="EXPERIMENTAL: Proxy to use for external build VMs in host:port format (without scheme). "
     "Ignored if --experimental-external is not enabled",
 )
+@click.option(
+    "--upload-cloud",
+    default="",
+    help="EXPERIMENTAL: Different cloud to use to upload the externally built image. The cloud "
+    "connection parameters should exist in the clouds.yaml. Ignored if --experimental-external is"
```

Comment:
Can you expand the help string to explain why `--experimental-external` is needed?

Reply:
Ok, would it be theoretically possible to use external upload without using an external builder?